CHANGELOG_OLD.md

```diff
@@ -251,7 +251,7 @@ _(none)_
 * Communicate detailed information about the difference between a resource's desired and actual state during a Pulumi update
 
 ## 0.18.6 (2019-07-04)
-* Upgraded to v1.5.0 of the Digital Ocean Terraform Provider
+* Upgraded to v1.5.0 of the DigitalOcean Terraform Provider
```

Comment:
These changes are harmless, but we use GH releases to comb the git log for release notes - this file is unused.